ap2/utils.py

```diff
@@ -79,7 +79,10 @@ def get_pycaw_volume_session():
 def get_volume():
     subsys = platform.system()
     if subsys == "Darwin":
-        pct = int(subprocess.check_output(["osascript", "-e", "output volume of (get volume settings)"]).rstrip())
+        try:
+            pct = int(subprocess.check_output(["osascript", "-e", "output volume of (get volume settings)"]).rstrip())
+        except:
```

Comment:
Please don't use a broad exception clause here. What is the exception raised that you are fixing?

Reply:
I think it's a ValueError